### ISSUE TITLE
No need to mark async fixtures with @pytest.mark.asyncio decorator

### ISSUE THREAD

**Issue by and-semakin** [m-burst/flake8-pytest-style#76]
# Rule request

## Description

Disallow using decorators `@pytest.fixture()` and `@pytest.mark.asyncio` at the same time.

## Rationale

[`pytest-asyncio`](https://github.com/pytest-dev/pytest-asyncio) allows defining asynchronous fixtures, like this:

```python
@pytest.fixture()
async def client(app: FastAPI) -> AsyncIterator[AsyncClient]:
    async with AsyncClient(app=app, base_url="http://test") as ac:
        yield ac
```

There is no need to put an additional mark on this fixture. But as the developer has to put `@pytest.mark.asyncio` mark on all the asynchronous tests, there is a chance that they'll put these redundant marks on fixtures as well:

```python
@pytest.fixture()
@pytest.mark.asyncio  # this mark is useless on fixtures and should be disallowed
async def client(app: FastAPI) -> AsyncIterator[AsyncClient]:
    async with AsyncClient(app=app, base_url="http://test") as ac:
        yield ac
```

Make sure that the order of decorators doesn't matter.

**Comment by m-burst** [comment]
Hi @and-semakin,
This and #75 is fixed in v1.4.0 which I've just released.

**Comment by and-semakin** [comment]
Awesome! Thank you!
